tests/unit/test_app_state_value.py

```diff
@@ -27,38 +30,69 @@ def dummy_callback_one_arg(new_value: Any) -> None:
 class DummyComponent():
     values = []
 
-    def callback_no_args(self) -> None:
+    def method_no_args(self) -> None:
         pass
 
-    def callback_one_arg(self, new_value) -> None:
+    def method_one_arg(self, new_value: Any) -> None:
         self.values.append(new_value)
 
+    def method_two_args(self, new_value: Any, other_value: Any) -> None:
+        self.values.append(new_value)
```

Comment:
Will the test ever use the appended _new\_value_ in the case of `method_two_args`? Leaving the line `self.values.append(new_value)` here, might evoke the impression there is a valid use-case for that, right?

Reply:
You're right! I'll fix it.